css/components.css

```diff
@@ -231,230 +232,230 @@ div .swiper-scrollbar-drag{
 	font-size: 10px;
 	line-height: 10px;
-	margin-bottom: 25px;
+	margin: 15px 0 25px;
 	display: inline-block;
 }
-@media (min-width: 480px){
-	.frm__btn{
+
+@media (min-width: 480px) {
+	.modal_form_button {
 		width: 300px;
 		height: 64px;
 		font-size: 13px;
 		line-height: 15px;
-		margin-bottom: 30px;
+		margin: 10px 0 20px;
 	}
 }
-@media (min-width: 768px){
-	.frm__btn{
+
+@media (min-width: 768px) {
+	.modal_form_button {
 		font-size: 15px;
 	}
 }
-@media (min-width: 1000px){
-	.frm__btn{
+
+@media (min-width: 1000px) {
+	.modal_form_button {
 		width: 330px;
 		height: 70px;
 	}
 }
-.frm__checkbox_container{
-	width: 245px;
-	margin: 0 auto;
-	text-align: start;
-}
-@media (min-width: 480px){
-	.frm__checkbox_container{
-		width: 300px;
-	}
-}
-@media (min-width: 1000px){
-	.frm__checkbox_container{
-		width: 330px;
-	}
+
+.modal_form__checkbox_container.input_container {
+	align-self: flex-start;
+	padding: 0 0 10px;
 }
-input.frm__checkbox_original{
+
+.modal_form__checkbox {
 	position: absolute;
 	z-index: -1;
 	opacity: 0;
 	width: 1.5em;
 	height: 1.5em;
 }
-.frm__label_checkbox{
+.modal_form__label_checkbox p{
+	text-decoration: underline;
+}
+.modal_form__checkbox+label {
 	font-size: 13px;
 	line-height: 15px;
 	color: #8A8A8A;
 	display: inline-flex;
 	align-items: center;
-	user-select: none;
 }
-@media (min-width: 480px){
-	.frm__label_checkbox{
+
+@media (min-width: 480px) {
+	.modal_form__checkbox+label {
 		font-size: 15px;
 		line-height: 18px;
 	}
 }
-:root{
-	--checkboxReg-color: #FFC700;
-}
-.frm__label_checkbox::before{
+
+.modal_form__checkbox+label::before {
 	content: "";
 	display: inline-block;
 	width: 15px;
 	height: 15px;
 	border: 2px solid;
-	border-color: var(--checkboxReg-color);
+	border-color: var(--color-yellow);
 	border-radius: 2px;
 	margin-right: 10px;
 	background-repeat: no-repeat;
 	background-position: center center;
 }
-@media (min-width: 480px){
-	.frm__label_checkbox::before{
+
+@media (min-width: 480px) {
+	.modal_form__checkbox+label::before {
 		width: 1.3333em;
 		height: 1.3333em;
 	}
 }
-.frm__checkbox_original:checked+label::before{
-	background-image: url(../img/questions/checkbox/x320/checkbox_x320.svg);
+
+.input_container.error .modal_form__checkbox+label::before {
+	border-color: var(--color-orange);
+}
+
+.modal_form__checkbox:checked+label::before {
+	background-image: url(../img/questions/checkbox/checkbox.svg);
+	background-size: contain;
+}
+
+.input_container {
+	position: relative;
+	align-self: center;
+	padding: 10px 0 5px;
 }
-@media (min-width: 480px){
-	.frm__checkbox_original:checked+label::before{
-		background-image: url(../img/questions/checkbox/x420/checkbox_x420.svg);
+@media (min-width: 1000px){
+	.input_container{
+		padding: 10px 0 10px;
 	}
 }
-.valid_tooltip{
+
+.valid_tooltip {
 	position: absolute;
-	color: #FF5C00;
+	color: var(--color-orange);
 	font-size: 10px;
 	line-height: 10px;
 	visibility: hidden;
 	opacity: 0;
 	transition: all 0.3s;
```

Comment:
always write exact properties you are going to animate, in order to avoid unpredictable situations and improve performance

Reply:
Done.